packages/amplify-util-mock/src/__e2e__/utils/lambda-helper.ts

```diff
@@ -0,0 +1,15 @@
+import * as path from 'path';
+import * as fs from 'fs-extra';
+
+export function getFunctionDetails(fnName: string) {
+    const lambdaFolder = path.join(__dirname, 'lambda_functions');
+    if(!fs.existsSync(path.join(lambdaFolder, `${fnName}.js`))) {
+        throw new Error(`Can not find lambda function ${fnName}`);
+    }
+
+    return {
+        packageFolder: lambdaFolder,
+        fileName: `${fnName}.js`,
```

Comment:
In ```api.ts``` we only allow index.js, how are these two are connected then? Folder name = function name, handler is always ```index.js``` currently, isn't it?

Reply:
When running test we don't provision resource like how we do in a project. Here the functions are put in a fixed folder `lambda_functions/<fn_name>.js`